README.markdown

```diff
@@ -150,6 +150,32 @@ API Implemented
 * [ngx.shared.DICT.replace](https://github.com/openresty/lua-nginx-module#ngxshareddictreplace)
 * [ngx.shared.DICT.delete](https://github.com/openresty/lua-nginx-module#ngxshareddictdelete)
 * [ngx.shared.DICT.flush_all](https://github.com/openresty/lua-nginx-module#ngxshareddictflush_all)
+* [ngx.shared.DICT.stats](#ngxshareddictstats)
+
+### ngx.shared.DICT.stats
+**syntax:** *total_used, total_size = ngx.shared.DICT:stats()
+
+**context:** *set_by_lua&#42;, rewrite_by_lua&#42;, access_by_lua&#42;, content_by_lua&#42;, header_filter_by_lua&#42;, body_filter_by_lua&#42;, log_by_lua&#42;, ngx.timer.&#42;, balancer_by_lua&#42;, ssl_certificate_by_lua&#42;, ssl_session_fetch_by_lua&#42;, ssl_session_store_by_lua&#42;*
+
+Retrieving the total used counts and total size in bytes for the dictionary [ngx.shared.DICT](#ngxshareddict).
+
+The first argument to this method must be the dictionary object itself, for example,
+
+```lua
+
+ local cats = ngx.shared.cats
+ local total_used, total_size = cats.stats(cats)
+```
+
+or use Lua's syntactic sugar for method calls:
+
+```lua
+
+ local cats = ngx.shared.cats
+ local total_used, total_size = cats:stats()
+```
+
+These two forms are fundamentally equivalent.
```

Comment:
It's better not elucidate the Lua method call syntax here. Otherwise we should add it to add to *all* the shdict methods' documentation, which is definitely not desired :)

Reply:
I copied it from [ngx.shared.DICT.set](https://github.com/openresty/lua-nginx-module#ngxshareddictset), but I noticed the other methods don't have this explanation.
So I deleted it all together.